generator/overrides/ticketing/pages/issuer/credential-form/index.tsx

```diff
@@ -1,15 +1,23 @@
-import { FC } from 'react'
+import { FC, useEffect } from 'react'
 import { Formik } from 'formik'
 
 import { Container, Header, Input, Textarea } from 'components'
 import { JSONLD_CONTEXT_URL } from 'utils/schema'
+import { messages } from 'utils/messages'
+
+import { notifyError } from 'utils/notification'
 
 import { initialValues, useCredentialForm } from './useCredentialForm'
 import * as S from './CredentialForm.styled'
 
-
 const CredentialForm: FC = () => {
-  const { handleSubmit, validate, isCreating } = useCredentialForm()
+  const { handleSubmit, validate, isCreating, error } = useCredentialForm()
+
+  useEffect(() => {
+    if (error) {
+      notifyError(new Error(messages.issuer.error.apiError))
```

Comment:
didn't change
and do not forget to add it as dependency to `useEffect`

Reply:
when added to useEffect, it says `unused dependency`